code/modules/client/preference/preferences.dm

```diff
@@ -2035,6 +2038,10 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	character.add_language(language)
 
+	if(species == "Grey")
+		if(speciesprefs)
+			character.speech_span = "wingdings"
```

Comment:
This won't apply if the character is cloned - you'll want to put this in the greys' `on_species_change` or something to set things up accordingly. Make sure to accordingly reset it when the player's species changes away from Grey, too.